workers/shared.py

```diff
@@ -86,20 +87,32 @@ def hdfs_to_local_path(hdfs_path):
 
     return filename
 
+def guess_ensembl_release(filepath):
+    try:
+        release = varcode.load_vcf(filepath)[0].ensembl.release
+    except ValueError:  # no guesses from varcode, return default
+        release = config.ENSEMBL_RELEASE
+    finally:
+        return release
 
 def load_vcf(vcf_path):
     """Return a vcf.Reader, header text for the given VCF."""
     if config.ALLOW_LOCAL_VCFS and vcf_path.startswith('/tests/'):
-        text = open(vcf_path[1:]).read()
+        filepath = vcf_path[1:];
+        text = open(filepath).read()
+        release = guess_ensembl_release(filepath)
```

Comment:
You can pull this `release = ...` out of the if/else block so you don't have to repeat it.